cli/build.go

```diff
@@ -65,12 +65,14 @@ func (b *BuildCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface{
 		if err != nil {
 			log.Errorf("Unable to parse argument: %v", err)
 			return subcommands.ExitFailure
-		}
-
-		pkg, err = ws.PackageByName(pkgName)
-		if err != nil {
-			log.Errorf("Unable to find package name %s: %v", pkgName, err)
-			return subcommands.ExitFailure
+		} else {
+			if pkgName != "" {
```

Comment:
pkgName can come out empty, if it does, we need to resolve it to the default value (name of directory, in other words, last component of `$PWD`)

Reply:
Yeah, it can come after the whole if-else bellow...